setup.py

```diff
@@ -53,8 +54,7 @@ def find_package_data(where='.', package='', exclude=standard_exclude, exclude_d
       url='https://courtformsonline.org',
       packages=find_packages(),
       namespace_packages=['docassemble'],
-      install_requires=['docassemble.AssemblyLine', 'docassemble.MassAccess', 'docassemble.PovertyScale', 'docassemble.ALToolbox>=0.7.1', 'docassemble.ALMassachusetts>=0.1.2',
-          'docassemble.GithubFeedbackForm>=0.2.1'],
+      install_requires=[],
```

Comment:
I'll revert this change, we do need these packages to be installed for the Affidavit to work.